app/views/layouts/application.html.erb

```diff
@@ -7,9 +7,18 @@
 
     <%= stylesheet_pack_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    
+    <!-- Tailwind-->
+      <meta charset="UTF-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <link href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css" rel="stylesheet">
+    <!-- Tailwind -->
   </head>
   <body>
+    <%= render "shared/header" %>
```

Comment:
When placing the render in the application.html.erb file, the bar is displayed from the home page.

![imagen](https://user-images.githubusercontent.com/12243331/102919685-eb415f80-444e-11eb-8a9e-e09ddaa7a9d8.png)

I recommend placing it on the page where you are working with modals.

Reply:
The requirements of this issue say that this navbar should be on all the pages of the app, is it true or not? Because if not, then I should only put the navbar with the modals as you said.